deep_qa/training/run_model.py

```diff
@@ -0,0 +1,117 @@
+
+import sys
+import logging
+import shutil
+import os
+
+import random
+import pyhocon
+import numpy
+
+# pylint: disable=wrong-import-position
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from deep_qa.common.params import Params, replace_none, ConfigurationError
```

Comment:
These can be relative imports, can't they?  And you shouldn't need the `sys.path.append` stuff.